ee/tabby-ui/lib/hooks/use-network-setting.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useQuery } from 'urql'
+
+import { graphql } from '../gql/generates'
+import { isClientSide } from '../utils'
+
+const networkSettingQuery = graphql(/* GraphQL */ `
+  query NetworkSetting {
+    networkSetting {
+      externalUrl
+    }
+  }
+`)
+
+const useQueryNetworkSetting = () => useQuery({ query: networkSettingQuery })
+
+const useNetworkSetting = () => {
```

Comment:
why we need both this and useQueryNetworkSetting? Can they be merged?

Reply:
Yes, for the convenience of using certain components, it seems unnecessary to use useNetworkSetting for now

Reply:
done